src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,383 @@
+		}
+
+		$update = $this->update_option( $option_name, $option_value );
+
+		if ( $update instanceof WP_Error ) {
+			/* translators: %1$s: option name, %2$s: error message */
+			WP_CLI::error( sprintf( __( 'Could not update %1$s option: %2$s', 'amp' ), $option_name, $update->get_error_message() ) );
+		}
+
+		/* translators: %s: option name */
+		WP_CLI::success( sprintf( __( 'Updated %s option.', 'amp' ), $option_name ) );
+	}
+
+	/**
+	 * Lists options and their values.
+	 *
+	 * This command can be used to list following options:
+	 * - AMP Plugin options.
+	 * - AMP Plugin options that can be updated via the CLI.
+	 * - Reader themes.
+	 *
+	 * ## OPTIONS
+	 * [<cli-managed-options>]
+	 * : List of options that can be updated via the CLI.
+	 *
+	 * [<reader-themes>]
+	 * : List of Reader themes.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * # List all options.
+	 * $ wp amp option list
+	 * +------------------+----------------+
+	 * | key              | value          |
+	 * +------------------+----------------+
+	 * | theme_support    | standard       |
+	 * | mobile_redirect  | disabled       |
+	 * | reader_theme     | legacy         |
+	 * +------------------+----------------+
+	 *
+	 * # List options that can be updated via the CLI.
+	 * $ wp amp option list cli-managed-options
+	 * theme_support, mobile_redirect, reader_theme
+	 *
+	 * # List Reader themes.
+	 * $ wp amp option list reader-themes
+	 * twentytwenty, twentytwentyone, legacy
+	 *
+	 * @subcommand list
+	 *
+	 * @param array $args Array of positional arguments.
+	 */
+	public function list_( $args = [] ) {
+		$user_cap = $this->check_user_cap();
+
+		if ( $user_cap instanceof WP_Error ) {
+			WP_CLI::error( $user_cap->get_error_message( 'amp_rest_cannot_manage_options' ) . PHP_EOL . WP_CLI::colorize( '%y' . $user_cap->get_error_message( 'amp_rest_cannot_manage_options_help' ) . '%n' ) );
+		}
+
+		if ( ! empty( $args ) ) {
+			list( $subcommand ) = $args;
+
+			if ( self::CLI_MANAGED_OPTIONS === $subcommand ) {
+				WP_CLI::line( implode( ', ', self::ALLOWED_OPTIONS ) );
+				WP_CLI::halt( 0 );
+			}
+
+			if ( self::READER_THEMES === $subcommand ) {
+				WP_CLI::line( implode( ', ', wp_list_pluck( $this->reader_themes->get_themes(), 'slug' ) ) );
+				WP_CLI::halt( 0 );
+			}
+
+			/* translators: %s: subcommand */
+			WP_CLI::error( sprintf( __( 'Invalid subcommand: %s', 'amp' ), $subcommand ) );
+		} else {
+			$options = $this->get_options();
+
+			if ( $options instanceof WP_Error ) {
+				/* translators: %s: error message */
+				WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+			}
+
+			WP_CLI\Utils\format_items(
+				'table',
+				array_map(
+					static function ( $option_name, $option_value ) {
+						return compact( 'option_name', 'option_value' );
+					},
+					array_keys( $options ),
+					$options
+				),
+				[ 'option_name', 'option_value' ]
+			);
+		}
```

Comment:
It may make sense to only every show the CLI-managed options, and then add a notice to list out the options that are managed via CLI.

Reply:
If someone wants to get all the raw options, they can always just do:

```
wp option get amp-options
```

Reply:
Updated the list command and it's more managed now 🎉 

![image](https://user-images.githubusercontent.com/54371619/216459860-af53edf4-5550-494d-8422-6ff6e12ec11b.png)